tests/integration/ci-runner.py

```diff
@@ -358,10 +358,15 @@ def _compress_logs(self, dir, relpaths, result_path):
         subprocess.check_call(  # STYLE_CHECK_ALLOW_SUBPROCESS_CHECK_CALL
             "sync", shell=True
         )
-        subprocess.check_call(  # STYLE_CHECK_ALLOW_SUBPROCESS_CHECK_CALL
+        retcode = subprocess.call(  # STYLE_CHECK_ALLOW_SUBPROCESS_CHECK_CALL
             "tar czf {} -C {} {}".format(result_path, dir, " ".join(relpaths)),
             shell=True,
         )
+        # tar return 1 when the files are changed on compressing, we ignore it
```

Comment:
Usually it shows that `clickhouse-server` is still working when `tar` is invoked, and this should be some issue in clickhouse (see also #28220)

Reply:
Still, it shouldn't completely fail the whole parallel running.

It's definitely a good idea to look for stale containers before the compressing. I just couldn't come with a quick fix in that place

Reply:
it look like it worked out in https://github.com/ClickHouse/ClickHouse/pull/44557